package.json

```diff
@@ -100,6 +100,7 @@
     "enzyme-to-json": "^3.3.4",
     "eslint-plugin-react-hooks": "^1.6.0",
     "jest": "^24.5.0",
+    "jest-styled-components": "^6.3.1",
```

Comment:
@brandonlilly just as a heads up, I merged my badge ticket to master yesterday which also adds `jest-styled-components`.